lib/electrumx_rpc/electrumx_client.dart

```diff
@@ -1113,17 +1115,23 @@ class ElectrumXClient {
         ],
       );
       try {
-        // If the response is -1 or null, return a temporary hardcoded value for
-        // Dogecoin.  This is a temporary fix until the fee estimation is fixed.
-        if (cryptoCurrency is Dogecoin &&
-            (response == null ||
-                response == -1 ||
-                Decimal.parse(response.toString()) == Decimal.parse("-1"))) {
-          // Return 0.05 for slow, 0.2 for average, and 1 for fast txs.
-          // These numbers produce tx fees in line with txs in the wild on
-          // https://dogechain.info/
-          return Decimal.parse((1 / blocks).toString());
-          // TODO [prio=med]: Fix fee estimation.
+        // If the response is -1 or null, fall back to the defaultFeeRate.
+        if (response == null ||
+            response == -1 ||
+            Decimal.parse(response.toString()) == Decimal.parse("-1")) {
+          if (CryptoCurrency is! BitcoinFrost) {
```

Comment:
`if (CryptoCurrency is! BitcoinFrost)` should be `if (cryptoCurrency is! BitcoinFrost)`